packages/app/src/cli/commands/app/logs.ts

```diff
@@ -51,10 +65,10 @@ export default class Logs extends Command {
 
     const sources = flags.source?.split(',')
 
-    await checkFolderIsValidApp(flags.path)
+    await checkFolderIsValidApp(flags.path!)
```

Comment:
Because of the `any` typecasting I added above, it thinks this can be either `string` or `undefined` so had to add the `!`.